Clicker/ViewControllers/CardController.swift

```diff
@@ -123,13 +123,13 @@ class CardController: UIViewController {
         countLabelBackgroundView.snp.makeConstraints { make in
             make.top.equalTo(view.safeAreaLayoutGuide.snp.top).offset(countLabelBackgroundViewTopPadding)
             make.centerX.equalToSuperview()
-            make.width.equalTo(countLabelWidth + countLabelHorizontalPadding * 2)
+            make.width.equalTo(countLabelWidth)
             make.height.equalTo(countLabelHeight)
         }
         
         countLabel.snp.makeConstraints { make in
             make.center.equalTo(countLabelBackgroundView.snp.center)
```

Comment:
```suggestion
            make.center.equalTo(countLabelBackgroundView)
```
Snapkit is cool bc it will infer the `.snp.cneter` on `countLabelBackgroundView`